### DISCUSSION TITLE
en/install/install-comfyui/install-comfyui-on-windows

### DISCUSSION THREAD

**Discussion by giscus** [comfyui-wiki/Comments#2] [Announcements]
# en/install/install-comfyui/install-comfyui-on-windows

Analysis of Common Installation Issues with ComfyUI

https://comfyui-wiki.com/en/install/install-comfyui/install-comfyui-on-windows

**Comment by comfyui-wiki** [comment]
This is a comment test. Hi everyone! comfyui-wiki.com is now support comments.

**Comment by elgigantegoesgit** [comment]
I installed it exactly acc to http://anakin.ai/blog/using-wan-2-1-with-comfyui-a-comprehensive-guide-for-windows-macos-and-linux/
on windows 11, python 3.11 (though py -v gives 3.13) but when i try to start, i get this error below
i do not find the "launch_windows.bat" file in the ComfyUI directory, so i do the following:

python.exe -s ComfyUI\main.py --windows-standalone-build

which gives

Checkpoint files will always be loaded safely.
Traceback (most recent call last):
  File "C:\dev\ComfyUI\main.py", line 136, in <module>
    import execution
  File "C:\dev\ComfyUI\execution.py", line 13, in <module>
    import nodes
  File "C:\dev\ComfyUI\nodes.py", line 22, in <module>
    import comfy.diffusers_load
  File "C:\dev\ComfyUI\comfy\diffusers_load.py", line 3, in <module>
    import comfy.sd
  File "C:\dev\ComfyUI\comfy\sd.py", line 6, in <module>
    from comfy import model_management
  File "C:\dev\ComfyUI\comfy\model_management.py", line 189, in <module>
    total_vram = get_total_memory(get_torch_device()) / (1024 * 1024)
                                  ^^^^^^^^^^^^^^^^^^
  File "C:\dev\ComfyUI\comfy\model_management.py", line 146, in get_torch_device
    return torch.device(torch.cuda.current_device())
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\dev\ComfyUI\venv\Lib\site-packages\torch\cuda\__init__.py", line 971, in current_device
    _lazy_init()
  File "C:\dev\ComfyUI\venv\Lib\site-packages\torch\cuda\__init__.py", line 310, in _lazy_init
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled

**Reply by comfyui-wiki** [reply]
Are you having problems with manual installation? For Windows, if you have an Nvidia graphics card, currently it is recommended to use ComfyUI Desktop for installation. The Desktop version will complete the installation of the corresponding environment and dependencies.

Currently, I am collaborating with the official Comfy team and have completed the content of the corresponding installation tutorial. You can refer to the following links:
- [ComfyUI Windows Desktop Version ( recommended )](https://docs.comfy.org/installation/desktop/windows)
- [ComfyUI Portable](https://docs.comfy.org/installation/comfyui_portable_windows)

The relevant installation tutorials on the [ComfyUI Wiki](https://comfyui-wiki.com/) will also be updated soon.
